l10n.yaml

```diff
@@ -1,7 +1,9 @@
 # Docs on this config file:
 #   https://docs.flutter.dev/ui/accessibility-and-localization/internationalization#configuring-the-l10nyaml-file
 
+synthetic-package: false
 arb-dir: assets/l10n
+output-dir: lib/generated/l10n
```

Comment:
Neat! Glad to see this fixed.

In addition to the points Chris mentioned above, let's add this directory to `.gitattributes` so that `git log -p` and so on don't become noisy; see the handling there of other generated files.